tutorials/multimodal_dapt_curation/curator/utils.py

```diff
@@ -0,0 +1,263 @@
+        """
+        text = text.replace("‘", "'").replace("’", "'")
+        text = text.replace("“", '"').replace("”", '"')
+        return text
+
+
+def clean_and_unify(dataset: DocumentDataset) -> DocumentDataset:
+    """
+    Cleans and unifies the given dataset using a set of predefined cleaners.
+
+    Args:
+        dataset (DocumentDataset): The dataset to be cleaned and unified.
+
+    Returns:
+        DocumentDataset: The cleaned and unified dataset.
+    """
+    cleaners = Sequential(
+        [
+            # Unify all the quotation marks
+            Modify(QuotationUnifier(), text_field="text"),
+            # Unify all unicode
+            Modify(UnicodeReformatter(), text_field="text"),
+        ]
+    )
+    return cleaners(dataset)
+
+
+def filter_text(dataset: DocumentDataset) -> DocumentDataset:
+    """
+    Filters the given dataset based on various criteria.
+    Refer to the full list of all filters here:
+    https://github.com/NVIDIA/NeMo-Curator/blob/main/config/heuristic_filter_en.yaml
+    https://github.com/NVIDIA/NeMo-Curator/blob/main/tutorials/peft-curation/main.py
+
+    Args:
+        dataset (DocumentDataset): The dataset to be filtered.
+
+    Returns:
+        DocumentDataset: The filtered dataset.
+    """
+    filters = Sequential(
+        [
+            # If a document contains a number of words not
+            # within a specified range then discard
+            ScoreFilter(
+                WordCountFilter(min_words=50, max_words=100000),
+                text_field="text",
+                score_field="word_count",
+                score_type=int,
+            ),
+            # If the document shrinks by > x% in terms of number of characters after
+            # removing the top n-grams then discard. Source: Gopher (Rae et al., 2021)
+            ScoreFilter(
+                RepeatingTopNGramsFilter(n=2, max_repeating_ngram_ratio=0.2),
+                text_field="text",
+                score_type=float,
+            ),
+            ScoreFilter(
+                RepeatingTopNGramsFilter(n=3, max_repeating_ngram_ratio=0.18),
+                text_field="text",
+                score_type=float,
+            ),
+            ScoreFilter(
+                RepeatingTopNGramsFilter(n=4, max_repeating_ngram_ratio=0.16),
+                text_field="text",
+                score_type=float,
+            ),
+            ScoreFilter(
+                RepeatedParagraphsFilter(max_repeated_paragraphs_ratio=0.7),
+                text_field="text",
+                score_type=float,
+            ),
+            # If more than 20% of the document is comprised of URLs then discard
+            ScoreFilter(
+                UrlsFilter(max_url_to_text_ratio=0.2),
+                text_field="text",
+                score_type=float,
+            ),
+        ]
+    )
+    filtered_dataset = filters(dataset)
+    return filtered_dataset
+
+
+def redact_PII(dataset: DocumentDataset) -> DocumentDataset:
```

Comment:
```suggestion
def redact_pii(dataset: DocumentDataset) -> DocumentDataset:
```
I prefer lowercase for this, but up to you.